tools/python/boutiques/tests/test_test.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import os, subprocess, pytest
+from unittest import TestCase
+from boutiques import __file__ as bfile
+from boutiques import bosh
+from jsonschema.exceptions import ValidationError
+
+class TestTest(TestCase):
+
+    def get_examples_dir(self):
+        return os.path.join(os.path.dirname(bfile),
+                            "schema", "examples")
+    
+    @pytest.mark.skipif(subprocess.Popen("type docker", shell=True).wait(), reason="Docker not installed")
```

Comment:
Just curious, why `type docker` over `which docker`? I don't think it matters, I've just always used `which` in this type of context :)

Reply:
I think @ValHayot introduced that in other parts of the tests. I had checked the exact same thing at that time: `type` is a bash built-in while `which` isn't, which makes the code more robust in case `which` is not installed. That said, it might crash with `sh`.

Reply:
Yeah, i had searched through stackoverflow to find the best command. Apparently missed `command -v` though, which would have us covered for everything: https://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script